dnf-plugins-core.spec

```diff
@@ -15,6 +15,12 @@
 %bcond_without python2
 %endif
 
+%if ! 0%{?rhel} || 0%{?rhel} > 7
```

Comment:
This check is wrong. It should be inverted:

```spec
%if 0%{?rhel} && 0%{?rhel} < 8
%bcond_with dnfutils
%else
%bcond_without dnfutils
%endif
```

Reply:
Why is it wrong? Your condition is just negation with switched branches. Or am I missing something?